tests/functional/smoke/full-compaction.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
```

Comment:
Make the year 2023, though I think we should just remove years all together. curl already went through the process of stripping them.

Reply:
```Under the 1978 US law, a copyright notice must contain the [copyright symbol](https://en.wikipedia.org/wiki/Copyright_symbol) (a lower case letter c completely surrounded by a circle) or its equivalent. The word "copyright" or the abbreviation "Copr." are also accepted in the US,[[11]](https://en.wikipedia.org/wiki/Copyright_notice#cite_note-11) but not in other countries. Works distributed outside the US use the © symbol. The copyright notice must also contain the year in which the work was first published (or created), and the name of the copyright owner, which may be the [author](https://en.wikipedia.org/wiki/Author) (including the legal author/owner of a [work made for hire](https://en.wikipedia.org/wiki/Work_made_for_hire)), one or more joint authors, or the person or entity to whom the copyright has been transferred
```

Including derivative works, which I suppose means whenever we change a file it's a derivative work???

https://en.wikipedia.org/wiki/Copyright_notice#:~:text=The%20copyright%20notice%20must%20also%20contain%20the%20year,entity%20to%20whom%20the%20copyright%20has%20been%20transferred.